src/snowflake/snowpark/modin/plugin/docstrings/series_utils.py

```diff
@@ -1121,6 +1121,38 @@ def nanosecond():
 
     @property
     def dayofweek():
```

Comment:
Those are [aliases](https://github.com/modin-project/modin/blob/b86028182d977d0c31c8fc96f34c62a3fbcc41d1/modin/pandas/series_utils.py#L643,L653) of `dayofweek` and `dayofyear`, so we only need to add docstrings to `dayofweek` and `dayofyear`.

Reply:
Yes, I know they are aliases. Having them equally covered by tests and docstrings is needed though. For docstrings, whatever version the user knows, they can search for it in the docs. This is what pandas does.

Reply:
`day_of_week` is `dayofweek`, so `day_of_week`'s docstring, `day_of_week.__doc__`, is `dayofweek`'s docstring, `dayofweek.__doc__`. Similarly for `dayofyear`. Adding docstrings to `day_of_week` and `day_of_year` in this file has no effect.

Reply:
Oh, ok. So are you saying that the net effect is that we will get two separate links to the same page in the docs similar to pandas? If that's the case then yes, we're good.

Reply:
Actually it feels like two different pages in pandas:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.dt.day_of_week.html
and
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.dt.dayofweek.html

Reply:
We should get two separate links, like pandas, because `dayofweek` and `day_of_week` are both attributes of `modin.pandas.series_utils.DatetimeProperties`. sphinx should generate two separate pages, as it does for modin ([dayofweek](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.dt.dayofweek.html) and [day_of_week](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.dt.day_of_week.html)).

Incidentally, Snowpark pandas documentation's [API reference](https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/latest/modin/index) doesn't seem to list any of the datetime accessor methods or properties.